CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Change Log
 
+## [0.21.0](https://github.com/eclipse/lemminx/milestone/34?closed=1) (June 29, 2022)
+
+### Enhancements
+
+ * Support folding for <!DOCTYPE. See [#1213](https://github.com/eclipse/lemminx/issues/1213).
+ * Improve XML formatter (option to have Tags in the same line). See [#594](https://github.com/eclipse/lemminx/issues/594).
+
+### Bug Fixes
+
+ * Incorrect error range for `cvc-identity-constraint.4.1`. See [#1210](https://github.com/eclipse/lemminx/issues/1210).
+ * Resolve uri as system with XML catalog. See [#1199](https://github.com/eclipse/lemminx/issues/1199).
+ * Preserve invalid content while formatting. See [#1041](https://github.com/eclipse/lemminx/issues/1041).
+ * Bad element formatting is replaced with <null>. See [#1034](https://github.com/eclipse/lemminx/issues/1034).
+ * Lone quote inside tag leads to catastrophic formatting. See [#679](https://github.com/eclipse/lemminx/issues/679).
+ * XML format for bad processing instruction removes following element. See [#675](https://github.com/eclipse/lemminx/issues/675).
+ * Fault tolerance needs improvement. See [#305](https://github.com/eclipse/lemminx/issues/305).
```

Comment:
Instead of using the issue name for this, describe the specific formatting issue that was addressed (I think its "attribute value without a key is removed")

Reply:
fixed